src/track/bpm.h

```diff
@@ -97,6 +97,21 @@ QDebug operator<<(QDebug dbg, const Bpm& arg) {
     return dbg << arg.getValue();
 }
 
+inline Bpm operator*(const Bpm& bpm, const int val) {
+    return (Bpm(bpm.getValue() * val));
+}
+
+inline Bpm operator/(const Bpm& bpm, const int val) {
```

Comment:
The integer division might not work as expected when passing a double. Use double here.